Add templates for standardization of files in each WG (#278)
chaoss/governance: templates/WG-README.md

> # CHAOSS \<_wg-name_\> Working Group

Reviewer: ```suggestion
# CHAOSS <wg-name> Working Group
```

Author: Backslashes are required else the word gets omitted, removed underscores

Badges (If Any)

## Table of Contents

- [Introduction](#introduction)
- [Participate](#participate)
- [Contributing](#contributing)
- [Metrics](#metrics)
- [Contributors](#contributors)
- [License](#license)

## Introduction

### Goals 

A brief description about the goals of the working group.
<br/>

### Purpose

Answers to why the working group was formed.
<br/>

### Who should join this working group?

Interests and requirements of the working group.
<br/>

### Background

- History of the working group
- Name changes in past (if any)
- Any other relevant past information
  
## Participate

### How to Join Us?

A section guiding the user on how to join the working group.

- Link to the Agenda/Meeting-Minutes document
- Link to the [Participate](https://chaoss.community/participate/) page on the website

Link to [Code of Conduct](https://github.com/chaoss/governance/blob/master/code-of-conduct.md)

## Contributing

> Link to `CONTRIBUTING.md` of the concerned working group.

Reviewer: ```suggestion
See the [CONTRIBUTING.md](CONTRIBUTING.md) for more info.
```

We know that file will exist.

## Metrics

### Focus-Areas and Goals

> Mainly a table of the focus areas and their goals linked to subfolders in the `focus-areas` folder. 

Reviewer: Idea: Add a draft table

> <br/>

Reviewer: ```suggestion
```

### Released Metrics
A section guiding the user to the released metrics.

- Link to the [metrics](https://chaoss.community/metrics/) page on the website

Link to the [translations](https://github.com/chaoss/translations) repository.

## Contributors

> ### Chairs

Reviewer: ```suggestion
### WG Maintainers
```

Author: As discussed in the meeting, we want to keep Chairs separate from the maintainers.

List containing names of contact persons (linked to their Github profiles) for the working group.

- 
- 

### Amazing CHAOSS Project Contributors

Link to [contributors](https://chaoss.community/metrics/#user-content-chaoss-contributors-include) listed on the website.

> <br/>

Reviewer: ```suggestion
```

## License

> - Link to the `LICENSE` file

Reviewer: ```suggestion
- See [LICENSE](LICENSE) file.
```

> - Copyright notice

Reviewer: Copyright (c) 201x - CHAOSS, a Linux Foundation Project

> <br/>

Reviewer: ```suggestion
```
